### DISCUSSION TITLE
Ability to insert to date / time in article that gets converted to users timezone

### DISCUSSION THREAD

**Discussion by cmgorton** [forem/forem#15470] [Icebox]
[Originally posted](https://forem.dev/rafi993/ability-to-insert-to-date-time-in-article-that-gets-converted-to-users-timezone-dk2) by Rafi on Forem.dev

It would awesome if you could insert date / time in article that shows in users timezone. This would be useful for users in different timezone especially for events. This could be achieved using a date time liquid tag or some formatting option.

**Describe the solution you'd like**
We could use liquid tag or some formatting option to show date / time in users timezone

**Additional context**

- Go to article with date / time example https://dev.to/azure/azurefunbytes-reminder-02-11-2021-2-pm-est-intro-to-azure-policy-with-stevenmurawski-f1k
- The article shows time in EST both in title and content
- But for users in different timezone they have to manually see what that time corresponds to
- But if there was a way to insert date / time that showed in users time zone it would awesome

**Comment by Sterling8426** [comment]
That really helpful. Recently I hired an expert for it so that he could insert date and time in the different post of our website about [sell my house](https://www.carolinashomebuyers.com/). And do you know that helped us alot in organic ranking. As our different articles have got good positions in ranking. And have shifted from the lower position in the serp. Now we are regularly adding it in our new articles.
